packages/dds/tree/src/tableSchema.ts

```diff
@@ -59,14 +59,12 @@ export namespace System_TableSchema {
 	 * @privateRemarks This interface primarily exists to provide a single home for property documentation.
 	 * @system @internal
 	 */
-	export interface OptionsWithSchemaFactory<
-		TScope extends string | undefined = string | undefined,
-	> {
+	export interface OptionsWithSchemaFactory<TSchemaFactory extends SchemaFactoryAlpha> {
```

Comment:
We're only interested in the schema factory itself here (and in a few other places), so we could just parameterize over that directly, rather than expressing the type in terms of the scope.